app/templates/network/show/details/partials/supplier.hbs

```diff
@@ -34,35 +34,36 @@
     </div>
   </div>
   <div class="row">
-  {{#tabbed-content active=activeTabDetails on-change="changeTabDetails" active-class="tab-button--active" as |manager|}}
+  {{#tabbed-content active=activeTabDetails on-change="changeTabDetails" active-class="tab-button--active" as |manager2|}}
       <div class="tabs__nav">
-      {{#tab-button class="tab-button"  manager=manager name="contracts"}}
+      {{#tab-button class="tab-button"  manager=manager2 name="contracts"}}
```

Comment:
Why is there a `manager` and a `manager2`? We need descriptive names for them, e.g. `sidebarTabManager` and `detailsTabManager` or similar.